viewer/packages/cad-parsers/index.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+export { CadMetadataParser } from './src/metadata/CadMetadataParser';
+export { MetadataRepository } from './src/metadata/MetadataRepository';
+export { CadModelMetadataRepository } from './src/metadata/CadModelMetadataRepository';
+export { CadModelMetadata } from './src/metadata/CadModelMetadata';
+export { SectorSceneFactory } from './src/utilities/SectorSceneFactory';
+export {
+  SectorScene,
+  BinaryFileProvider,
+  File3dFormat,
+  ModelDataClient,
+  BlobOutputMetadata
```

Comment:
Seems to only be used outside cad-parsers so I dont think it belongs in this package.

Reply:
I see that ModelDataClient is used by `cad-parsers/src/metadata/CadModelMetadataRepository.ts`, and that it again relies on BinaryFileProvider and JsonFileProvider.. BlobOutputMetadata can be removed, I think